docs/high-availability/api-server/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM gcr.io/google_containers/kube-apiserver:9680e782e08a1a1c94c656190011bd02
```

Comment:
Some things were Wacky with gcr, not sure why I had to rebuild the images locally . Obviously not related to the containers themselves